WordPress/Classes/ViewRelated/Post/PostSettingsViewController.m

```diff
@@ -764,6 +777,16 @@ - (UITableViewCell *)configureGeolocationCellForIndexPath:(NSIndexPath *)indexPa
     return cell;
 }
 
+- (UITableViewCell *)configureExcerptCellForIndexPath:(NSIndexPath *)indexPath
+{
+    UITableViewCell *cell = [self getWPTableViewCell];
```

Comment:
True that. In this case, with ARC, we'd be 100% fine, but i did miss there are a bunch of other `getXXX` methods as well.

Agreed, let's just limit to the initial scope, and swiftify later!